gensrc/script/doris_builtins_functions.py

```diff
@@ -1360,6 +1360,10 @@
         [['negative'], 'DECIMAL64', ['DECIMAL64'], ''],
         [['negative'], 'DECIMAL128', ['DECIMAL128'], ''],
 
+        [['oct'], 'VARCHAR', ['BIGINT'], ''],
+        [['oct'], 'VARCHAR', ['VARCHAR'], ''],
```

Comment:
as you have not supported varchar/string input now, dont put them here

Reply:
fixed